### DISCUSSION TITLE
第2节课笔记(4班)

### DISCUSSION THREAD

**Discussion by shiruo2686** [InternLM/Tutorial#82] [笔记]
4班的第2节课笔记，请提交到这里！

**Comment by shiruo2686** [comment]
释若第2次课笔记：https://blog.csdn.net/weixin_60280805/article/details/135433073

**Comment by zhjunqin** [comment]
笔记：https://blog.csdn.net/zhjunqin/article/details/135433000?spm=1001.2014.3001.5502

**Comment by onlyzer0** [comment]
笔记：https://blog.csdn.net/qq_42907812/article/details/135445094?spm=1001.2014.3001.5501
onlyzer0
